study.md

```diff
@@ -41,7 +41,7 @@ Does `proxima_centauri.print_distance` run? Why or why not? If so, then what is
 the output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
+<!--They will not print because the attributes are private.-->
```

Comment:
It runs since `@distance_km` is an instance variable and within the scope of the function.